src/NetherPortalScanner.h

```diff
@@ -39,8 +39,8 @@ class cNetherPortalScanner : public cChunkStay
 	static const int SearchSolidBaseWidth = 3;
 
 	/** Where to place the player out from the face and across the face */
-	const double m_OutOffset = 2;
-	const double m_AcrossOffset = 0.5;
+	const double OutOffset = 2;  // NOLINT
+	const double AcrossOffset = 0.5;  // NOLINT
```

Comment:
Just use the `static` keyword, it shouldn't trigger clang-tidy.

Reply:
Yeah, but than it fails to build for me :/

Reply:
I forgot that you need to add a setting for 
```
readability-identifier-naming.ConstantMember
```
and that will override the `PrivateMember` setting when it's marked `const`.

Reply:
> and that will override the `PrivateMember` setting when it's marked `const`

But should this be allowed?:
```
class A
{
    const int x;
}
```

If not, then we should use `readability-identifier-naming.ClassConstantPrefix` to only match `static` class constants.

And note, that you need also need to set the case. Otherwise Clang-Tidy will ignore an empty prefix:
```
- key:             readability-identifier-naming.ClassConstantPrefix
  value:           ''
- key:             readability-identifier-naming.ClassConstantCase
  value:           'aNy_CasE'
```